src/main/java/com/powsybl/openloadflow/sensi/AbstractSensitivityAnalysis.java

```diff
@@ -98,6 +104,12 @@ protected JacobianMatrix createJacobianMatrix(EquationSystem equationSystem, Vol
     }
 
     static class LfSensitivityFactor<T extends EquationTerm> {
+
+        enum Status {
+            VALID,
+            SKIP,
+            NULL
```

Comment:
You don't need this enum value, just use a null status

Reply:
For me, null and Status.NULL would represent two different things:
- If the status is Status.NULL, it means that the value for this sensitivity factor will be null (0), because it is a sensitivity on a disconnected branch
- if the status is null (meaning: the sensitivity factor has no value), then this is a bug

Reply:
So here NULL means zero. this make me think it is NULL as null in Java so an enum with a mix of business and technical values which is a bad practice